### PR TITLE
schema amends: change boolean selects to checkboxes 

### PR DESCRIPTION
**Pull request by moloko**
see adaptlearning/adapt_authoring#1380

**please leave unmerged until all core plugins have been updated**

### CODE REVIEW

File: js/assessment.js [hunk 1]
```diff
@@ -242,7 +242,7 @@ define([
 
         get: function(id) {
             if (id === undefined) {
-                return this._assessments.slice(0);
+                return this._assessments[0];
```
Comment (oliverfoster) [inline js/assessment.js]: this isn't a minor change

Comment (brian-learningpool) [inline js/assessment.js]: Good catch @oliverfoster.  I didn't actually need a `getFirst()`.  The change in 4ec4121 seems to do enough.

Comment (oliverfoster) [inline js/assessment.js]: I still don't think the approach you've taken solves the problem you've described.

https://github.com/adaptlearning/adapt-contrib-assessmentResults/blob/master/js/adapt-contrib-assessmentResults.js#L225
```javascript
var assessmentModel;
if (!state.id) {
assessmentModel = Adapt.assessment.get()[0];
} else {
assessmentModel = Adapt.assessment.get(state.id);
}
```
^ if no id defined in assessmentResults, pick the first one.

https://github.com/adaptlearning/adapt-contrib-assessment/blob/master/js/adapt-assessmentArticleModel.js#L108
```javascript
var assessmentConfig = this.get("_assessment");
if (!assessmentConfig._id) {
    assessmentConfig._id = this.model.get("_id");
}
```
^ if no id defined in assessment, take article id

**Comment by moloko** [comment]
Sorry but I do have to say this has now got well beyond the bounds of what should be 'tacked on' to a a PR that is specifically to do with schema changes! 

@brian-learningpool would you mind setting up an issue that documents the problem you are trying to solve and creating a new PR to go with it?

**Comment by brian-learningpool** [comment]
Issue raised here:
https://github.com/adaptlearning/adapt_framework/issues/1621

**Comment by brian-learningpool** [comment]
Closing PR at @moloko's request.  This has been split into two separate PRs here:
- https://github.com/adaptlearning/adapt-contrib-assessment/pull/91 
- https://github.com/adaptlearning/adapt-contrib-assessment/pull/92

**Pull request closed without merging**